python_wrapper/susi_python/main.py

```diff
@@ -160,10 +160,14 @@ def add_device(access_token):
         session = device_info['session'] # session info
         identity = session['identity']
         name = identity['name']
+        curr_location = geocoder.ip('me')
         params2 = {
         'macid': macid,
         'name': name,
+        'room': 'Living-Room',
```

Comment:
Do note that in the future, we should allow users to specify where the device is located instead of predefining it in the living room.

Reply:
Yes @cweitat . That will be passed through the mobile Apps